ex8/js/updatePhoneBook.js

```diff
@@ -0,0 +1,50 @@
+function insertJson(employeeList) {
+  // Remove ol element
+  document.getElementById("phonebook").remove();
+
+  // Create new ol element
+  var oLelement = document.createElement('ol');
+  oLelement.setAttribute('id', 'phonebook');
+  document.body.appendChild(oLelement);
+
+  var el, elSpan, elLink;
+  for (var i = 0; i < employeeList.length; i++) {
+    el = document.createElement('li');
+
+    elSpan = createSpan("name", employeeList[i].name);
+    el.appendChild(elSpan);
+
+    elLink = createLink("email", employeeList[i].email);
+    elSpan = createSpan("email");
+    elSpan.appendChild(elLink);
+    el.appendChild(elSpan);
+
+    elLink = createLink("phone", employeeList[i].phone);
+    elSpan = createSpan("phone");
+    elSpan.appendChild(elLink);
+    el.appendChild(elSpan);
+
+    // Add employee list to ol
+    document.getElementById('phonebook').appendChild(el);
+  }
+}
+
+function createSpan(className, info, link) {
+  var element = document.createElement('span');
+  var attribute = document.createAttribute("class");
+  attribute.value = className;
+  element.setAttributeNode(attribute);
+  if (info !== undefined) {
+    element.innerHTML = info;
+  }
+  return element;
+}
+
+function createLink(link, info) {
+  var element = document.createElement('a');
+  var attribute = document.createAttribute("href");
+  attribute.value = link + ":" + info;
+  element.setAttributeNode(attribute);
+  element.innerHTML = info;
+  return element;
+}
```

Comment:
You should put these functions in the `exports` or `module.exports`. Right now they are globally available and used, only if you `require` them. This is called a side-effect and it's bad.
if you put these in `exports`, the caller can then be:
`require('updatePhoneBook', function(updatePhoneBook) { updatePhoneBook.insertJson()); }`